source/command-monitoring/tests/unified/command.yml

```
@@ -0,0 +1,75 @@
+description: "commands"
```

For consistency with the file name.

```suggestion
description: "command"
```